apps/internal/base/base.go

```diff
@@ -89,6 +89,7 @@ type AuthResult struct {
 	ExpiresOn      time.Time
 	GrantedScopes  []string
 	DeclinedScopes []string
+	IsFromCache    bool
```

Comment:
Please update some unit tests to cover this. I am fairly sure there are tests that cover retrieval from cache / network.

Reply:
I updated the test that checks for cache now also checks for `IsFromCache` to `true`
and likewise when we do a noncached check it checks for `IsFromCache` to `false`